### ISSUE TITLE
MissingMethodExcepiton with Target Framework net461

### ISSUE THREAD

**Issue by MarkusBrgr** [domaindrivendev/Swashbuckle.AspNetCore#741]
Because I want to host my AspNetCore Application in a Windows Service, i changed the target Framework from netcore2.0 to net461. Now when calling the SwaggerUI it throws following Exception:

```
fail: Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware[0]
      An unhandled exception has occurred while executing the request
System.MissingMethodException: Methode nicht gefunden: "Void Microsoft.Net.Http.Headers.EntityTagHeaderValue..ctor(System.String)".
   bei Microsoft.AspNetCore.StaticFiles.StaticFileContext.LookupFileInfo()
   bei Microsoft.AspNetCore.StaticFiles.StaticFileMiddleware.Invoke(HttpContext context)
   bei Microsoft.AspNetCore.HttpOverrides.HttpMethodOverrideMiddleware.<Invoke>d__4.MoveNext()
--- Ende der Stapelüberwachung vom vorhergehenden Ort, an dem die Ausnahme ausgelöst wurde ---
   bei System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   bei System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   bei Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.<Invoke>d__7.MoveNext()
```

Everything else works fine, just Swagger won't show up.

**Comment by berndku** [comment]
We have the same issue. Also the Swagger UI does not display with net641 and ASPNET.Core

**Comment by kennywangjin** [comment]
We have the same issue when targeting net47 with AspNetCore.

**Comment by Starkie** [comment]
We managed to solve this error by updating the package to the [4.0.1](https://github.com/domaindrivendev/Swashbuckle.AspNetCore/releases/tag/v4.0.1) release.
Everything working as expected now.

**Comment by domaindrivendev** [comment]
As per @Starkie's response, I'm assuming this to be resolved as of 4.0.1.
